src/seismometer/seismogram.py

```diff
@@ -280,9 +285,7 @@ def _load_from_config(self, config_path: str | Path):
         self.predict_time = self.config.predict_time
         self.output_list = self.config.output_list
         self.target_event = self.config.target
-
-        self._load_metadata()
-        self._load_data()
+        self._cohorts = self.config.cohorts
```

Comment:
Rather than setting _cohorts, should we just redirect to self.config as a property?

Reply:
I started to do this but will defer with the same expectation as the cohort_cols.

This attribute doesn't seem to be accessed currently, indicating some more focused attention is probably warranted

Reply:
See #32